File: client/src/app/core/models/classes/Match.ts
import { IMatch } from "../interfaces/IMatch";

export class Match implements IMatch {
	id: number;
	idTeam1: number;
	idTeam2: number;
	stt: number;
	round: number;
	created: Date;
	startAt: Date;
	tournamentId: number;

	constructor() {
		this.id = 0;
		this.idTeam1 = 0;
		this.idTeam2 = 0;
		this.stt = 0;
		this.round = 0;
		this.created = new Date();
		this.startAt = new Date();
		this.tournamentId = 0;
	}
}

File: core/CreateTournament/Models/Match.cs
using System.Collections.ObjectModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace CreateTournament.Models
{
    public class Match : DeleteableEntity
    {
        public int IdTeam1 { get; set; }
        public int IdTeam2 { get; set; }
        public DateTime Created {  get; set; } = DateTime.UtcNow ;
        public DateTime StartAt { get; set; } = DateTime.UtcNow;
        public int round { get; set; }
        public int TournamentId { get; set; }
        [ForeignKey("TournamentId")]
        public Tournament Tournament { get; set; }
    }
}
